policy/lsst/sensorNameToFieldIdx.yaml

```diff
@@ -2,9 +2,10 @@
 
 # The mapping between the sensor name and field index
 
-R00_S12: 33
-R00_S21: 33
-R00_S22: 33
+R00_SG0: 33
+R00_SG0: 33
+R00_SW0: 33
+R00_SW1: 33
```

Comment:
The file of `sensorNameToFieldIdx.yaml` is used to get the related elements in the sensitivity matrix. One "pair" of wavefront sensor has only one field point. Therefore, it might be better to just use: `R00_SW` here and modify the related code in the calculation.

You know, each pair of corner wavefront sensors (one extra-focal and one intra-focal CCDs) only gives one Zernike polynomial calculation. This is why we only consider one field point here.

Reply:
It might be:

```
R00_SG0: 33
R00_SG1: 33
R00_SW: 33
```

or

```
R00_SG1: 33
R00_SG0: 33
R00_SW: 33
```

Reply:
Ah, ok, I will go with a single `SW` point then.